Xamarin.Forms.Core.UnitTests/AbsoluteLayoutTests.cs

```diff
@@ -196,7 +196,7 @@ public void SizeRequestWithRelativeSizeChild ()
 
 			var sizeReq = abs.GetSizeRequest (double.PositiveInfinity, double.PositiveInfinity);
 
-			Assert.AreEqual (new Size (210, 60), sizeReq.Request);
+			Assert.AreEqual (new Size (110, 40), sizeReq.Request);
```

Comment:
and with this one ?

Reply:
as above, but because the offset is not proportional, it is added to the size (100 + 10, 20 + 20)